symfony/src/Controller/SitesController.php

```diff
@@ -42,20 +42,20 @@ public function edit(SiteRepository $siteRepository, EntityManagerInterface $em,
         if ($id) {
             $site = $siteRepository->find($id);
             if (!$site) {
-                throw $this->createNotFoundException('Page notice not found.');
+                throw $this->createNotFoundException('Site not found.');
             }
 
             if ($request->request->has('delete')) {
                 $em->remove($site);
                 $em->flush();
                 $loggerService->log(Log::SITE_DELETE, $site->getId());
-                $this->addFlash('success', 'Site removed');
+                $this->addFlash('success', 'Site removed.');
                 return $this->redirectToRoute('admin_sites');
             }
         } else {
             if ($syncEnabled) {
                 // can't create new sites if syncing from rdr
-                throw $this->createNotFoundException('Page notice not found.');
+                throw $this->createNotFoundException('Site not found.');
```

Comment:
Can you change this to something like `Sites cannot be created when RDR sync is enabled.` as requested in the ticket?